README.rst

```diff
@@ -312,6 +312,13 @@ Example usage:
         '/static/custom-admin-theme.js',
     ]
 
+``OPENWISP_REST_SWAGGER``
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**default**: ``True``
+
+Enable or disable Swagger API documentation endpoint.
```

Comment:
I would specify how to access the swagger API and that to enable it is required to include the urls in the root urlconf, eg: ` url(r'^api/v1/', include('openwisp_utils.api.urls'))`